recipes/example/meta.yaml

```diff
@@ -80,4 +80,4 @@ extra:
     # GitHub IDs for maintainers of the recipe.
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
-    - LandoCalrissian
+    - LandoCalrissian
```

Comment:
This is minor, but could you ensure that there are **no changes** to this file?

Reply:
Never touching this file again hahaha.
Thank you.